### PR TITLE
upgrade node version

### PR DESCRIPTION
**Pull request by dkent600**
Upgrades to 11.15.0, the highest version that builds.  I spent a lot of time trying to get 12.x to work, without ultimate success.  I think 11.15.0 accomplishes the spirit of the request.

Removes the `cheap` devtool in the prod webpack config and still builds.

### CODE REVIEW

**Comment by dkent600** [comment]
> what issues were you seeing with 12?

Errors all relating to node-gyp, scrypt and node-boilerplate.  Some package or packages are out-of-sync.

**Comment by dkent600** [comment]
> yes, 11 might be better that 12 ..
> though we must have the ability to build alchemy with the latest recommended node . 12.6.1.
> 
> which action (or dependencies)are required to solve it ?

@orenyodfat I am proposing to do this now and worry about 12 in a separate issue.  11 takes care of our immediate needs.

**Comment by orenyodfat** [comment]
> > yes, 11 might be better that 12 ..
> > though we must have the ability to build alchemy with the latest recommended node . 12.6.1.
> > which action (or dependencies)are required to solve it ?
> 
> @orenyodfat I am proposing to do this now and worry about 12 in a separate issue. 11 takes care of our immediate needs.

why ? what are our immediate needs ?  which action (or dependencies)are required to solve it ?

**Comment by dkent600** [comment]
> why ? which immediate needs it is solving? 

This upgrade was originally motivated by needing to solved the problem with the size of the heroku slug.  You, I and heroku support (see the [most recent comment here](https://help.heroku.com/tickets/837443#event_fa9027b8-99d4-4937-9cea-75eabc8d0311)) all recommended upgrading beyond v10 as possibly the best way to do this.  At the moment we are working-around the problem with a potentially fragile hack.

Upgrading node to version 11 is all we need to obtain memory reductions that may reduce the fragility of the work-around and maintain our position below the slug size limit.  We don't need version 12 to obtain these memory reductions.

This is not a huge thing since we are already below the slug size limit.  But upgrading to version 12 has even less immediate benefit.

I have already spent a over a day trying to get v12 to build after two weeks of working on nothing but super boring infrastructure issues.  I would like a break from that.

**Comment by dkent600** [comment]
> why ? which immediate needs it is solving? 

@orenyodfat 

This upgrade was originally motivated by needing to solved the problem with the size of the heroku slug.  You, I and heroku support (see the [most recent comment here](https://help.heroku.com/tickets/837443#event_fa9027b8-99d4-4937-9cea-75eabc8d0311)) all recommended upgrading beyond v10 as possibly the best way to do this.  At the moment we are working-around the problem with a potentially fragile hack, a hack that also _significantly diminishes our ability to debug the preview apps_.  We would like to eliminate that hack.

Upgrading node to version 11 may all by itself be able to obtain memory reductions sufficient to enable us to eliminate the hack.  We don't need version 12 to obtain these memory reductions.

I have already spent a over a day trying to get v12 to build after two weeks of working on nothing but super boring infrastructure issues.  I would like a break from that.

**Comment by orenyodfat** [comment]
`Upgrading node to version 11 may all by itself be able to obtain memory reductions sufficient to enable us to eliminate the hack. We don't need version 12 to obtain these memory reductions.` 

please remove the cheap-source-maps patch and confirm that torus review app can be build without an issue.

File: package.json [hunk 1]
```diff
@@ -10,8 +10,8 @@
     "url": "git+https://github.com/daostack/alchemy.git"
   },
   "engines": {
-    "node": "10.16.0",
-    "npm": "6.11.3"
+    "node": "11.15.0",
+    "npm": "6.7.0"
```
Comment (orenyodfat) [inline package.json]: why npm is downgraded ?

Comment (dkent600) [reply package.json]: Probably because that is the minimum version that will work with node 11.15.0.

**Comment by dkent600** [comment]
@orenyodfat Forgive me, in my comment above I related the node version to the slug size problem, but in fact is related to the problem running out of memory when building the review app for the Torus PR.

File: package.json (hunk 1 above)
Comment (orenyodfat) [reply package.json]: but before we used 6.11.3 . why not keeping that ?

Comment (dkent600) [reply package.json]: Because this is better for our users, gives them more options.

**Comment by dkent600** [comment]
It is looking like using node v11.15.0 fixes the problem without having to change the sourceMap devtool.

**Pull request merged**